cpp/src/fil/fil.cu

```diff
@@ -500,61 +484,58 @@ void tl2fil_leaf_payload(fil_node_t* fil_node, const tl::Tree::Node& tl_node,
 }
 
 void node2fil_dense(std::vector<dense_node_t>* pnodes, int root, int cur,
-                    const tl::Tree& tree, const tl::Tree::Node& node,
+                    const tl::Tree& tree, int node_id,
                     const forest_params_t& forest_params) {
-  if (node.is_leaf()) {
+  if (tree.IsLeaf(node_id)) {
```

Comment:
Pascal-style capitalization? Is that a DMLC thing?

Reply:
Yes, DMLC uses modified Google Style, which includes Pascal-style capitalization. For some reason only the Tree object was not using it and I made it consistent with the rest of Treelite project.

Reply:
Seems fair!